Rel 11.7.0.0

sjb017 opened the pull request.

sjb017 committed:
PSM3 OFI Provider from IEFS 11_6_0_1_1

Ensure 11.6.0.1 release matches SRPM for IEFS release

Signed-off-by: Scott Breyer <scott.breyer@intel.com>

sjb017 committed:
PSM3 OFI Provider from IEFS 11_7_0_0_110

Ensure 11.7.0.0 release matches SRPM found in IEFS release

Signed-off-by: Scott Breyer <scott.breyer@intel.com>

acgoldma commented on VERSION:
```diff
@@ -1 +1 @@
-11_6_0_0
+11_6_0_1
```
11_7_0_0 should be in the 2nd patch.

sjb017 replied:
Will fix

sjb017 committed:
PSM3 OFI Provider from IEFS 11_7_0_0_110

Ensure 11.7.0.0 release matches SRPM found in IEFS release

- Update VERSION file

Signed-off-by: Scott Breyer <scott.breyer@intel.com>

sjb017 committed:
PSM3 OFI Provider from IEFS 11_7_0_0_110

Ensure 11.7.0.0 release matches SRPM found in IEFS release

- Handle deleted files

Signed-off-by: Scott Breyer <scott.breyer@intel.com>
